README.md

```diff
@@ -14,27 +14,109 @@ The [draft site](https://draft-openlibertyio.mybluemix.net/) is built using the
 
 # Publishing a new release of Open Liberty Docs
 
-This is the flow that must be followed when releasing a new version of docs on openliberty.io. These steps update the `vNext`, `staging` and `draft` branches of the [docs repo](https://github.com/OpenLiberty/docs) so that the next version of docs can be worked on and ensures that fixes can still be made to the docs of the version being released.
+This is the flow that must be followed when releasing a new version of docs on openliberty.io. These steps update the `vNext`, `staging` and `draft` branches of the [docs repo](https://github.com/OpenLiberty/docs) so that the next version of docs can be worked on and ensures that fixes can still be made to the docs of the version being released. This procedure assumes you have already cloned the [docs](https://github.com/OpenLiberty/docs), [docs-generated](https://github.com/OpenLiberty/docs-generated), and docs-playbook repos.
+
+The following sections provide instructions to publish an Open Liberty release from the command line. For similar instructions that use the GitHub desktop client, see [Publishing a new release of Open Liberty Docs by using GitHub desktop](https://github.com/OpenLiberty/docs-playbook/edit/draft/README.md#Publishing-a-new-release-of-Open-Liberty-Docs-by-using-GitHub-desktop)
+
+## Updating the generated docs
+
+The Open Liberty generated docs, which comprise the majority of the **Features** and **Server configuration** sections of the doc, are built from the [docs-generated repo](https://github.com/OpenLiberty/docs-generated). For each Open Liberty release, Chuck Bridgham from the kernel team posts the updated generated docs to the `draft` branch of the docs-generated repo. Contact Chuck several days before the release to ensure the generated docs are updated on draft in time to publish. 
+
+Once you have verified the generated doc on the draft site, open a PR from `draft` to `staging` in the docs-generated repo and get it reviewed and merged. Once youy verify the generated doc on the `staging` site, open a PR from `staging` to `vNext` in the docs-generated repo. Get the PR reviewed and merge it in.
+
+## Publishing the docs and preparing for the next release
 
 Before proceeding, make sure that there are no current builds in the `Build` step of the [prod pipeline](https://cloud.ibm.com/devops/pipelines/fcc7c3e9-9c40-4a58-8a7f-09c08413ab7d?env_id=ibm:yp:us-south). This is so merging pull requests later into prod does not impact any current deployments.
```

Comment:
@mbroz2 - does this still apply? We generally  disregard it since none of us have access to the IBM Cloud pipeline- seems like it might apply more for the website admins who kick off the build at the end. 

But when the kickoff gets automated, would this be something we'd need to confirm with an admin before we started the publishing process?

Reply:
I think this can be removed

Reply:
fixed